src/deployment/flavors.md

```diff
@@ -3,41 +3,158 @@ title: Creating flavors for Flutter
+named `development`. 
+The build configurations in the development scheme 
+have the -development suffix. 
+
+3. Duplicate the build configurations to differentiate between the 
+default configurations that are already available and the new configurations 
+for the `development` scheme. Under the **Info** tab at the end of the 
+**Configurations** dropdown list, click the plus button and duplicate 
+each configuration name(Debug, Release, and Profile). 
+Duplicate the listed production configurations, once for each environment.   
+
+![Step 3 Xcode image](/assets/images/docs/flavors/step3-ios-build-config.png){:width="100%"}
+
+4. To match the development flavor, add -development 
+at the end of each new configuration name. 
+
+5. Change the `development` Scheme to match the development 
+build configurations you’ve created to assign the correct build configurations 
+to the `development` scheme. 
+* In the Runner project, click Manage Schemes… a pop up window opens. 
+* Double click the development scheme. In the next step 
+(as shown in the screenshot), you’ll modify each scheme 
+to match its dev build configuration:
+
+![Step 5 Xcode image](/assets/images/docs/flavors/step5-ios-scheme.png){:width="100%"}
+
+## Using flavors in iOS
+
+Now that you’ve set up your development flavor, 
+you can add different product bundle identifiers per flavor. 
+A bundle identifier is a way to uniquely identify your application. 
+In this example, we want the Debug-development value to equal 
+com.flavor-test.dev. 
+
+1. Change the app bundle identifier to differentiate between schemes. 
+In **Product Bundle Identifier**, append `.dev` to each -development scheme value.            
+![Step 1 using flavors image.](/assets/images/docs/flavors/step1-using-flavors.png){:width="100%"}        
+2. In the Build Settings, set the Product Name value to match each flavor.        
+![Step 2 using flavors image.](/assets/images/docs/flavors/step2-using-flavors.png){:width="100%"}             
+3. Add the display name to info.plist.        
+Create a new key in info.plist called **Bundle Display Name** 
+with the value `$(PRODUCT_NAME)`. 
+![Step 2 using flavors image.](/assets/images/docs/flavors/step2-using-flavors.png){:width="100%"}    
+
+Now you have set up your development flavor by making a `development` scheme 
+in Xcode and setting the build configurations for that scheme. 
+
+For more information, skip to the [“Launching your app flavors”][] 
+section at the end of this document.
+
+## Using flavors in Android
+
+Setting up flavors in Android can be done in your projects 
+**build.gradle** file.
+
+1. In your IDE, inside your Flutter project, 
+navigate to **android** > **app** > **build.gradle**.   
+
+![Step 1 using flavors in Android image.](/assets/images/docs/flavors/android-step1.png){:width="100%"}    
+
+Add a flavors object with the specified environments along with values for 
+dimension, resValue, and applicationId. 
```

Comment:
Should flavors be plural if you're adding a single object?

```suggestion
Add a flavors object with the specified environments along with values for 
**dimension**, **resValue**, and **applicationId**. 
```

Reply:
Thanks for pointing that out.